src/data.json

```diff
@@ -171,6 +258,11 @@
     "text": "€€€€, Club & Disco",
     "main_image_path": "https://s3-media0.fl.yelpcdn.com/bphoto/bow4ZZU5X0Y6Fj4fsMuAYA/348s.jpg",
     "beschreibung": "Darkflower Alternative Club Leipzig- Das Darkflower ist Treff der schwarzen Szene. Der bekannteste schwarze Szene-Club bietet alles von Alternative bis EBM.",
-    "öffnungszeiten": "Montag 10 - 18 Uhr Dienstag 10 - 18 Uhr Mittwoch 10 - 18 Uhr Donnerstag 10 - 18 Uhr"
+    "öffnungszeiten": [
+      "Montag 10 - 18 Uhr",
+      "Dienstag 10 - 18 Uhr",
+      "Mittwoch 10 - 18 Uhr",
+      "Donnerstag 10 - 18 Uhr"
+    ]
```

Comment:
I think you’d save some lines of you could group the object into a smaller chunk. Maybe as a single object that you call? Not sure though.